tests/test_docopt.py

```diff
@@ -708,22 +715,226 @@ def test_issue_71_double_dash_is_not_a_valid_option_argument():
 usage: pit stop"""
 
 
-def test_parse_section():
-    assert parse_section("usage:", "foo bar fizz buzz") == []
-    assert parse_section("usage:", "usage: prog") == ["usage: prog"]
-    assert parse_section("usage:", "usage: -x\n -y") == ["usage: -x\n -y"]
-    assert parse_section("usage:", usage) == [
```

Comment:
@h4l It looks like the constant `usage` isn't used anymore, should be deleted

Reply:
Ah yes, I'll get rid of it.

Reply:
- [ ] remove unused usage constant